src/builder.rs

```diff
@@ -93,11 +141,12 @@ mod test {
         signature: Signature,
         f: impl FnOnce(FunctionBuilder<true>) -> Result<BuildHandle<FuncID<true>>, BuildError>,
     ) -> Result<Hugr, BuildError> {
-        let mut module_builder = ModuleBuilder::new();
+        let mut builder = HugrBuilder::new();
+        let mut module_builder = builder.module_builder();
         let f_builder = module_builder.declare_and_def("main", signature)?;
 
         f(f_builder)?;
 
-        module_builder.finish()
+        builder.finish()
```

Comment:
Again, do we need to call `module_builder.finish()`? (Probably I'm missing something.)